webclient/pas/migrations/0011_alter_samostatnynalez_projekt.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.1.5 on 2025-03-05 11:21
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pas', '0010_alter_samostatnynalez_igsn_and_more'),
+        ('projekt', '0010_projekt_pristupnost_snapshot'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='samostatnynalez',
+            name='projekt',
+            field=models.ForeignKey(db_column='projekt', limit_choices_to={'typ_projektu': 1135}, on_delete=django.db.models.deletion.RESTRICT, related_name='samostatne_nalezy', to='projekt.projekt'),
```

Comment:
@jhavrlant Tato migrace je vyloženě nebezpečná, protože dané ID v `imit_choices_to={'typ_projektu': 1135}` bude platit jen na jednom ze serverů. Nevím, co vše to ovlivňuje, ale může to být potenciálně problém